src/Parser/VengeanceDemonHunter/CPM_ABILITIES.js

```diff
@@ -13,14 +13,6 @@ export const SPELL_CATEGORY = {
 
 const CPM_ABILITIES = [
```

Comment:
Please move this to a new CastEfficiency module, see the HolyPaladin spec for an example (it's fairly trivial, mostly a matter of copying an example, moving the 2 arrays and adding the module to your specModules).